src/main/java/co/cask/hydrator/plugin/hive/action/HiveImport.java

```diff
@@ -43,15 +43,6 @@ public HiveImport(HiveConfig config) {
 
   @Override
   public void configurePipeline(PipelineConfigurer pipelineConfigurer) throws IllegalArgumentException {
-    //validate hive command. For import we only accept Load statements
-    if (!config.statement.toUpperCase().startsWith("LOAD")) {
-      throw new IllegalArgumentException("To import data to a hive table, please use Hive LOAD query");
-    }
-
-    if (config.statement.length() < 15) {
```

Comment:
since you removed this check, can the next statement (now on line 47) throw a `StringIndexOutOfBoundsException`?